gcs/CONFIGURATION.md

```diff
@@ -152,6 +152,15 @@ provider.
 
     The implementation of the AccessTokenProvider interface used for GCS
     Connector.
+    
+*   `fs.gs.refresh.access-token.per.request.enable` (default: `false`)
+
+    Whether to generate a new access token for each GCS request.
+    Note: When setting this to true, you have to have an AccessTokenProvider 
+    implementation provided. When using `refresh.access-token.per.request.enable`
+    The low level HTTP retry will be turned off.
+    Note: When this flag is set to true, it takes precedency over 
+    `fs.gs.authorization.handler.impl`
```

Comment:
Why do we need this option?

Reply:
Probably this is a naming issue, this option is really about "Should we obtain a token that is related to the request in hand".

Reply:
Converted this to an enum type and renamed to `authorization.mode`